src/Tooltip/Tooltip.tsx

```diff
@@ -2,12 +2,13 @@ import React from 'react';
 import Tippy from '@tippyjs/react';
 import 'tippy.js/dist/tippy.css';
 
-type TootipProps = {
+type TooltipProps = {
   children: React.ReactElement;
-  tooltipContent: React.ReactElement;
+  content: React.ReactElement;
+  interactive: boolean;
 };
-export const Tooltip: React.FC<TootipProps> = ({ tooltipContent, children, ...rest }) => (
-  <Tippy {...rest} content={tooltipContent}>
+export const Tooltip: React.FC<TooltipProps> = ({ interactive, content, children, ...rest }) => (
+  <Tippy interactive={interactive} {...rest} content={content}>
```

Comment:
Ah ok, I see what is going on now. Do you think `interactive` needs to be explicit? I think we we just needed to add ti to the story to allow it to happen.